controller/frontend/src/Controller/Frontend/Customer/Standard.php

```diff
@@ -74,7 +78,7 @@ public function __clone()
 	 * Creates a new customer item object pre-filled with the given values but not yet stored
 	 *
 	 * @param array $values Values added to the customer item (new or existing) like "customer.code"
-	 * @return \Aimeos\Controller\Frontend\Customer\Iface Customer controller for fluent interface
+	 * @return Iface Customer controller for fluent interface
```

Comment:
Was that intentional?

Reply:
I think phpstorm did some magic, i reverted all import simplifications back to the full path.